### ISSUE TITLE
Update TypeScript definition to support v6.X

### ISSUE THREAD

**Issue by thinkingserious** [sendgrid/sendgrid-nodejs#428]
#### Issue Summary

v6.X is a major breaking change from v5.X, so the TypeScript definition needs an update.

**Comment by thinkingserious** [comment]
cc @SPARTAN563

**Comment by SPARTAN563** [comment]
Will take a look, can't promise that I'll get to it this week with SRECon, however I'll do what I can.

**Comment by thinkingserious** [comment]
No worries @SPARTAN563, enjoy the conference!

**Comment by bogacg** [comment]
Meanwhile SPARTAN563's PR gets reviewed, we can trick things a bit to work with Typescript:
```
// SINCE WE DONT HAVE TYPINGS READY FOR '@sendgrid/mail'
// REQUIRE w. any WILL  BE USED

/// <amd-dependency path="model/tree" />
declare var require: (moduleId: string) => any;
var sgMail = require('@sendgrid/mail')();
```

**Comment by thinkingserious** [comment]
Thanks @bogacg!

**Comment by thinkingserious** [comment]
Solved with #430
